src/row_codec.rs

```diff
@@ -47,26 +49,38 @@ pub(crate) fn encode_row_v0(
     value: Option<&[u8]>,
     row_features: &[RowFeature],
     timestamp: Option<i64>,
+    expire_at: Option<i64>,
 ) {
     data.put_u16(key_prefix_len as u16);
     data.put_u16(key_suffix.len() as u16);
     data.put(key_suffix);
 
-    data.put(encode_meta(row_features, value.is_none(), timestamp));
+    data.put(encode_meta(
+        row_features,
+        value.is_none(),
+        timestamp,
+        expire_at,
+    ));
 
     if let Some(value) = value {
         data.put_u32(value.len() as u32);
         data.put(value);
     }
 }
 
-fn encode_meta(row_features: &[RowFeature], is_tombstone: bool, timestamp: Option<i64>) -> Bytes {
+fn encode_meta(
+    row_features: &[RowFeature],
+    is_tombstone: bool,
+    timestamp: Option<i64>,
+    expire_at: Option<i64>,
+) -> Bytes {
     let mut meta = BytesMut::new();
 
     for attr in row_features {
         match attr {
             RowFeature::Flags => meta.put_u8(encode_row_flags(is_tombstone)),
             RowFeature::Timestamp => meta.put_i64(timestamp.expect("Timestamp RowAttribute was enabled for SST but attempted to insert a row with no timestamp.")),
+            RowFeature::ExpireAtTs => meta.put_i64(expire_at.unwrap_or(NO_EXPIRE_TS))
```

Comment:
yeah we can do that 👍 (though remember that they specify TTL, not `expire_ts` and we compute `expire_ts` based on `ttl + now`, where `now` is returned from the clock) so for this to return `i64::MAX` would be pretty weird (like their clock, which we enforce monotonicity, always returning `i64::MAX` and `ttl` always returning `0`)

Reply:
I'll change it to `i64::MIN` since that'll be basically impossible to accidentally generate (it would mean you have a clock that always returns `i64::MIN` and ttls that always return 0)